docs/book/src/configuration/squid-directives.md

```diff
@@ -0,0 +1,1826 @@
+# Squid Directives
```

Comment:
These are a lot of rules and "directives". Also I would not mention squid at all in these documents as it's just an inspiration not really meant as a specific spec to follow.

That project most likely also didn't start with a giant list of directives, but grew over time I imagine.

I would much rather want to just start to come up with a small list of rules that proofs the value of such a config, and that can already be supported by a v0.2 or v0.3 of rama.

Not sure if I make much sense though.